resources/lang/nb.json

```diff
@@ -0,0 +1,395 @@
+{
+    "Actions": "Handlinger",
+    "Details": "Detaljer",
+    "If you did not request a password reset, no further action is required.": "Hvis du ikke forespurte tilbakestilling av passord, ingen videre handlinger er nødvendig",
+    "Reset Password": "Tilbakestill passord",
+    "Sorry! You are not authorized to perform this action.": "Beklager! Du er ikke autorisert til å utføre denne handlingen.",
+    "You are receiving this email because we received a password reset request for your account.": "Du mottar denn ee-posten fordi vi har mottat en forespørsel om å tilbakestille passordet til din konto.",
+    "Confirm Password": "Bekreft passord'",
+    "Dashboard": "Dashboard",
+    "Email Address": "E-post Adresse",
+    "Forgot your password?": "Glemt ditt passord?",
+    "Forgot your password?":"Glemt ditt passord?",
+    "Login":"Logg inn",
+    "Logout":"Logg ut",
+    "Password":"Passord",
+    "Remember Me":"Husk Meg",
+    "Resources":"Ressurser",
+    "Send Password Reset Link":"Send Passord Tilbakestillings Link",
+    "Welcome Back!":"Velkommen Tilbake!",
```

Comment:
Translations for keys with "Title Case" in English should use "Sentence case" in other languages. "Velkommen tilbake!", "Slett ressurs", "Frakoble valgte" etc.